source/frontend/src/features/documents/documentDetail/DocumentDetailContainer.tsx

```diff
@@ -47,7 +47,7 @@ export const DocumentDetailContainer: React.FunctionComponent<
 
   const formikRef = useRef<FormikProps<DocumentUpdateFormData>>(null);
 
-  const deleteModalProps = getCancelModalProps();
+  const deleteModalProps = useMemo(() => getCancelModalProps(), []);
```

Comment:
whats this about? Was it doing an extra load?

Reply:
infinite render loop on my local.